python/common/python/math.java

```diff
@@ -0,0 +1,52 @@
+package python;
+
+@org.python.Module(
+        __doc__ = "A module that supplies some mathematical functions.\n"+
+                    "Current functions:\n"+
+                    "pow(x, y) -> Return x raised to the power y as type float."
+)
+
+public class math extends org.python.types.Module {
+    public math() {
+        super();
+    }
+
+    @org.python.Method(
+        __doc__ = "Returns x raised to the power y.\n",
+        args = {"x", "y"}
```

Comment:
Maybe change variable names to something more explanatory, for example base and exponent.